script/Ciel/Day 9/09_01_CIEL09_1B.txt

```diff
@@ -116,65 +129,74 @@ The moment I saw you, I suddenly felt like playing a prank..."
 [sha:69ced862bbd2dd45fe4af769525e1e599538693d]{
 -- Page 49, Offset 22257.
 -- 「いえいえ、朝から元気な事は結構です。このぐらいの狼藉、普段の行いに比べれば大した事はありませんし。
-"No, no, I'm glad you're so full of energy this early in the morning. Besides, this act of savagery doesn't amount to much compared to what you usually do.
+Oh, don't worry about it. I'm happy to see you're in such high spirits this morning.
+ Besides, this level of tomfoolery doesn't amount to much compared to your usual behavior.
 }
 [sha:86a35fe1a016195471fd55bb998c81c300be4559]{
 -- Page 49, Offset 22258.
 -- おはようございます遠野くん。
-Good morning to you, Tohno-kun.
+Good morning, Tohno-kun.
 }
 [sha:5f071a95b34d1f0cc6e6c107c91bebcd50991b90]{
 -- Page 49, Offset 22259.
 -- 無茶ばかりして、体と頭は大丈夫ですか？」
-You've been straining yourself so much, both physically and mentally. Are you okay?"
+You've really been burning the candle at both ends recently.
+ How are you holding up, both physically and mentally?
 }
 [sha:0b9f0a08fb7b25f2db83ef4ff63d77fa7586174e]{
 -- Page 50, Offset 22260.
 -- 先輩は極上の笑顔で極上の挨拶をしてくれた。
-Senpai offers me a sumptuous smile that goes with her sumptuous greeting.
+Senpai offers me a sumptuous smile to go with her sumptuous greeting.
 }
 [sha:caf5797fb4690688462de8e92edf9e54c69b4e0a]{
 -- Page 50, Offset 22261.
 -- こっちも「おはようございます、先輩」と返す。
-I reply 'Good morning, Senpai'.
+I wish her a pleasant morning as well.
 }
 [sha:12c42cfe6d30b8545f42620f505d3b16c602adc4]{
 -- Page 50, Offset 22262.
 -- なんか笑顔に異質なプレッシャーを感じるけど気にしない。先輩の笑顔はいつだって最高だ。
-I do feel a dissonant pressure in her smile but that doesn't trouble me. Senpai's smile is wonderful as always.
+I feel like there's an unusual pressure behind her smile, but I don't really pay it any mind.
+ Any smile of Senpai's is a wonderful smile.
 }
 [sha:0beefd6762bc83cc81ae704dc0423d12b8213c61]{
 -- Page 51, Offset 22263.
 -- 「もうすぐホームルームなので失礼しますね。
-"Homeroom's starting soon, so you'll have to excuse me.
+"Homeroom's about to start, so you'll have to excuse me.
 }
 [sha:8b706c48fdd11a1aa29056ce0d5e7d242ffc2ce7]{
 -- Page 51, Offset 22264.
 -- あ、そうだ。お昼休み、茶道室に来てください。是非お話ししたい事があるので。
-Ah, also. Please come to the tea ceremony room when lunch comes around. I have some things that I absolutely need to talk to you about.
+Oh, that reminds me.
+ Would you mind coming to the tea ceremony during lunch?
```

Comment:
is there a reason 'room' was removed?

Reply:
![aichanteehee](https://user-images.githubusercontent.com/2476799/197915363-bb91aaa8-49ef-4dde-81b4-f741a8747134.png)